pub_sub/go/http/checkout/app.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func main() {
+	var DAPR_HOST, DAPR_HTTP_PORT string
+	var okHost, okPort bool
+	if DAPR_HOST, okHost = os.LookupEnv("DAPR_HOST"); !okHost {
+		DAPR_HOST = "http://localhost"
+	}
+	if DAPR_HTTP_PORT, okPort = os.LookupEnv("DAPR_HTTP_PORT"); !okPort {
+		DAPR_HTTP_PORT = "3500"
+	}
+	const PUBSUB_NAME = "order_pub_sub"
+	const PUBSUB_TOPIC = "orders"
+	for i := 1; i <= 10; i++ {
+		order := "{\"orderId\":" + strconv.Itoa(i) + "}"
+		client := &http.Client{}
```

Comment:
Avoid using `&` at the site of declaration. Instead, use it when you _share_ it. Also, I know this is an example, but people cut and paste examples into production code. The default HTTP client config us not suitable to production.

https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779